### DISCUSSION TITLE
Reduce Time Series interval

### DISCUSSION THREAD

**Discussion by rofiaini** [gmtsar/gmtsar#909] [Q&A]
Hai all, I want to ask, if I have done the processing until I get intf_all and want to reduce the time interval, can I immediately use the interferogram in intf_all or do I have to do the processing from the start?

**Comment by Xiaohua-Eric-Xu** [comment]
You may want to create a new intf.in file and difference the current one with previous one to figure out what new interferograms to be generated. Then run intf_tops.csh to get the new interferograms processed.
